packages/pages/package.json

```diff
@@ -109,7 +111,17 @@
     "jest-environment-jsdom": "^28.1.3",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "ts-jest": "^28.0.7"
+    "ts-jest": "^28.0.7",
+    "@storybook/react": "^6.5.10",
+    "@storybook/addon-links": "^6.5.10",
+    "@storybook/addon-essentials": "^6.5.10",
+    "@storybook/addon-actions": "^6.5.10",
+    "@storybook/addon-interactions": "^6.5.10",
+    "@storybook/testing-library": "^0.0.13",
+    "@storybook/builder-webpack4": "^6.5.10",
+    "@storybook/manager-webpack4": "^6.5.10",
+    "@babel/core": "^7.18.10",
```

Comment:
Is babel necessary?

Reply:
@mkilpatrick There's an [issue on Github](https://github.com/storybookjs/storybook/issues/12826#issuecomment-713478017) where it looks like you can use a pre-built fork, or disable passing the components through babel (though it would still be a dependency). It's only a dev dependency so we could leave it, but I understand wanting to limit that too.

What are your thoughts on this?

Reply:
Nah it's fine. Didn't realize what it was needed for.